### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by RubenYoo**

### CODE REVIEW

File: test_knight_piece.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+def test_knight_get_valid_peaceful_moves():
+    """
+    Unit tests for the get_valid_peaceful_moves method of the Knight class
+    """
+
+    # global instance of the game_state
+    my_game_state = chess_engine.game_state()
+
+    # =============== Boundary Values Test 01 ==================
+
```
Comment (talbarfm) [inline test_knight_piece.py:15]: Separate each test to a different function
